configs/sparse_rcnn/sparse_rcnn_r50_fpn_300_proposals_crop_mstrain_480-800_3x_coco.py

```diff
@@ -0,0 +1,49 @@
+_base_ = './sparse_rcnn_r50_fpn_mstrain_480-800_3x_coco.py'
+num_proposals = 300
+model = dict(rpn_head=dict(num_proposals=num_proposals))
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+
+train_pipeline = [
```

Comment:
May add a comment indicating that this augmentation strategy originates from DETR.